packages/fxa-auth-server/docs/swagger/password-api.ts

```diff
@@ -129,13 +129,26 @@ const PASSWORD_FORGOT_STATUS_GET = {
   ],
 };
 
+const PASSWORD_CREATE_POST = {
+  ...TAGS_PASSWORD,
+  description: '🔒 sessionToken',
+  notes: [
+    dedent`
+      🔒 HAWK-authenticated with session token
+
+      Creates a new password for the user associated with the session token. Creating a new password will generate new encryption key.
```

Comment:
Might be worth noting this is only for passwordless accounts. (having these docs is already awesome)